plugin/evm/customethclient/ethclient_ext.go

```diff
@@ -0,0 +1,70 @@
+// (c) 2019-2025, Ava Labs, Inc. All rights reserved.
```

Comment:
this can be renamed to `ethclient.go` without the `_ext` suffix

Reply:
let's keep it as it's actually extending the ethclient